utilities/workers/worker_manager.py

```diff
@@ -26,7 +26,8 @@ def create(
         input_queues: "list[queue_proxy_wrapper.QueueProxyWrapper]",
         output_queues: "list[queue_proxy_wrapper.QueueProxyWrapper]",
         controller: worker_controller.WorkerController,
-    ) -> "tuple[bool, WorkerProperties]":
+        main_logger: logger,
```

Comment:
Rename parameter to `local_logger` , since it's not guaranteed that this logger is from main. Do similar for below.